toolchain_deps/Cargo.sgx_trusted_lib.toml

```diff
@@ -76,3 +77,4 @@ wasmi             = { git = "https://github.com/mesalock-linux/wasmi-sgx" }
 webpki            = { git = "https://github.com/mesalock-linux/webpki", branch = "mesalock_sgx" }
 webpki-roots      = { git = "https://github.com/mesalock-linux/webpki-roots", branch = "mesalock_sgx" }
 yasna             = { git = "https://github.com/mesalock-linux/yasna.rs-sgx" }
+rusty_machine     = { git = "https://github.com/mesalock-linux/rusty-machine-sgx", package="rusty-machine"}
```

Comment:
What is `package="rusty-machine"`? I guess it's not needed.

All sgx packages can be declared same as https://github.com/mesalock-linux/crates-sgx/blob/3e5c6eae9b70063fd891a776cd7bfb6a585c359f/Cargo.toml#L76.